Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -165,6 +165,13 @@ class Frontend {
   /// Read and prepare the file at the given path for processing, optionally synchronizing
   /// diagnostic output.
   private func openAndPrepareFile(at url: URL) -> FileToProcess? {
+    guard !IgnoreFile.isStandardIgnoreFile(url) else {
+      diagnosticsEngine.emitError(
```

Comment:
Could you add a test case that ensures this diagnostic gets produced?

Reply:
Do you have existing code to test the front end output over a directory?
If you do, then sure.
If not, I think adding that code is outside the scope of this change and would be better off as a follow-up issue.